modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -132,13 +132,32 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
       renderTypeAlias(shapeId, ta.nameRef, tpe, recursive, hints)
     case enumeration @ Enumeration(shapeId, _, values, hints) =>
       renderEnum(shapeId, enumeration.nameRef, values, hints)
-    case _ => Lines.empty
+  }
+
+  private def deprecationAnnotation(hints: List[Hint]): Line = {
```

Comment:
the only thing I'd change here is have an extra map, before the that's already there where I call `renderStringLiteral` before building the `line` to avoid `${renderStringLiteral(msg)}`

Reply:
tbh I actually like that `renderStringLiteral` is being interpolated (as long as it doesn't break the line)